ApiTransform.ps1

```diff
@@ -394,7 +426,7 @@ $TransformConfig = [Ordered]@{
             'workdays_put'                                      = 'Workday_Set';
             'workdays_workers'                                  = 'WorkdayWorker_Get';
         };
-        ExcludedList       = @('/applications/{application_id}') #, '/applications/{application_id}/logo', '/logos/{id}')
+        ExcludedList       = @('/applications/{application_id}', '/applications/{application_id}/import/users', '/providers/{provider_id}/integrations', '/providers/{provider_id}/integrations/connectwise', '/integrations/connectwise/{UUID}', '/integrations/connectwise/{UUID}/agreements', '/integrations/connectwise/{UUID}/agreements/{agreement_ID}/additions', '/integrations/connectwise/{UUID}/companies', '/integrations/connectwise/{UUID}/mappings', '/providers/{provider_id}/administrators', '/providers/{provider_id}/administrators/{id}', '/providers/{provider_id}/organizations', '/administrators/{id}/organizationlinks', '/administrators/{administrator_id}/organizationlinks/{id}', '/organizations/{id}/administratorlinks') #, '/applications/{application_id}/logo', '/logos/{id}')

```

Comment:
If possible it would be nice to see a reason for why we excluded each of these.

Reply:
Besides the /Applications/ endpoints the remainder of these functions are MTP only functions. We'll need to re-work how / when the ORG ID is required when running those endpoints.